core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_coreos

```diff
@@ -1,5 +1,7 @@
+quay.io/centos/centos:centos7 registry.ci.openshift.org/coreos/centos:centos7
 quay.io/centos/centos:8 registry.ci.openshift.org/coreos/centos8:8
 quay.io/centos/centos:stream8 registry.ci.openshift.org/coreos/stream8:8
+quay.io/centos/centos:centos8 registry.ci.openshift.org/coreos/centos:centos8
```

Comment:
How is this tag different from the one on L2?

Reply:
Not entirely sure the difference but it looks like the tag `centos:centos8` is older than the `centos:8` in [quay.io](https://quay.io/repository/centos/centos?tab=tags). 

We need to test something with an older centos image. I think the thought is to see if this older centos8 image works before going all the way back to centos7. Suspicion is that we have to back to centos7 because that was the last known working image though.